main_logic/core.py

```
@@ -2937,10 +2965,31 @@ async def tts_response_handler(self):
+                                    self._tts_respawn_task = None
+                            else:
+                                logger.warning("⚠️ 收到TTS未就绪信号，13秒后尝试重新拉起Worker")
+                                # 取消之前的延迟重试任务（如有）
+                                if self._tts_respawn_task and not self._tts_respawn_task.done():
+                                    self._tts_respawn_task.cancel()
+                                    self._tts_respawn_task = None
+                                async def _delayed_respawn():
+                                    await asyncio.sleep(13)
+                                    if not self.is_active or self.tts_ready:
+                                        return
+                                    logger.info("🔄 TTS 延迟重试：尝试重新拉起 Worker...")
```

Ruff (0.15.7)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)